core/src/modules/configLoaders/property.ts

```diff
@@ -55,8 +56,8 @@ export async function loadProperty(
     await property.setFilters(configObject.filters, externallyValidate);
   }
 
-  if (configObject.identifying === true) {
-    await property.makeIdentifying();
+  if (configObject["identifying"] !== null) {
```

Comment:
```suggestion
  if (!!configObject["identifying"]) {
```

This change would ensure that the value is nil (either `undefined` or `null`)